teacher.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Teacher < Person
+  attr_reader :specialization
+
+  def initialize(specialization, age, name = 'Unknown')
+    super(age, name)
```

Comment:
- [ ] The parameters here should be the parameters given in the `Person` class.e.g. `super(age, name, parent_permission)`. Kindly fix this.